<?xml version="1.0" encoding="UTF-8"?>
<launch>
 
    <node pkg="cones_perception" type="cone_detection" name="cone_detection" output="screen"/>
    <node pkg="cones_perception" type="color_classifier_server" name="color_classifier_server" output="screen"/>

</launch>